### ISSUE TITLE
Noob here - can anyone change the title of message box or tell me how to?

### ISSUE THREAD

**Issue by feetsdr** [broken-e/DragDropConfirm#26]
I can see the 'Hold up there...' text in the FileDragDropExt.cpp file.  I'd like to change that to a more Windows / office setting phrase , maybe just 'Move confirmation' or similar.

I can copy that file to my machine and edit that file, but don't know what to do after that : ) 

From what little I know of software development, that file (and others?) need to be compiled?  I downloaded a couple different things - visual studio, et al. But didn't really know what to do after that. And started wondering if it has to be compiled with other files?

Any advice or if it's stupidly easy to edit / do what's needed to get the installer to use the 'Move Confirmation' version, I'd appreciate it : )

**Comment by TerraD** [comment]
Yes, that is possible. @bduffek has implemented this and other (more important) goodies!

The title is set by:
[HKEY_LOCAL_MACHINE\SOFTWARE\DragDropConfirm]
"AskTitle"="Hold up there..."

Look for all details in: https://github.com/broken-e/DragDropConfirm/pull/13

Please note: you should have version v1.3.0.0 of the .dll. There is no package as there used to be. But simply download the two .dll and the installer from https://github.com/bduffek/DragDropConfirm/tree/01e14bc1581fc52bd0455bd6ea0491ee49185e42/Installer to a folder of your choice and run the installer.

Have fun!

**Comment by feetsdr** [comment]
Thanks for the info!  Brace yourself.... I said I am a noob!!

1) I tried going there in the registry.  But under software, there isn't a dragdropconfirm key.  Is what you are describing an override?

I did search the registry for dragdropconfirm and there are keys for it. but none with asktitle or askdescription values like on the #13 page.

Are those only there if I type them myself?

I checked programfiles / dragdropconfirm.  There's 4 files in there.  The 32 and 64 bit DLLs.  Both are 1.3.0.0, a readme and uninstall.

I was going to download the installer again from github.  

From https://github.com/bduffek/DragDropConfirm

I clicked on 'goto file and get a page with lots of files.  

I guess I am supposed to click on download zip.

THat did it.  I reran the installer.  still no asktitle in the registry.  There's default text that comes (in the installer? in the DLLs?).  Is there a way to change there, rather than need to run a reg file to add the keys?

**Comment by feetsdr** [comment]
And following your link -  https://github.com/bduffek/DragDropConfirm/tree/01e14bc1581fc52bd0455bd6ea0491ee49185e42/Installer

How would I download those three files from that page?

**Comment by TerraD** [comment]
I think you have to download them one by one. Left-click the first, then use Download-Button and so on....

**Comment by feetsdr** [comment]
OK.  How about the [HKEY_LOCAL_MACHINE\SOFTWARE\DragDropConfirm] ?

That key isn't there.  
a) On a machine that DDC is already installed, I added that key and some of the entries from #13.   And then DDC didn't work : ) 

I change the name of the key and DDC starts working.  This is what is in the key:

ey Name:          HKEY_LOCAL_MACHINE\SOFTWARE\DragDropConfirm
Class Name:        <NO CLASS>
Last Write Time:   8/20/2021 - 10:04 AM
Value 0
  Name:            AskTitle
  Type:            REG_SZ
  Data:            fdsfdfdsfsd

Value 1
  Name:            AskDescription
  Type:            REG_SZ
  Data:            fgsafdsafsdafdsaf

Value 2
  Name:            ItemText
  Type:            REG_SZ
  Data:            dsfdsafdsaf

Value 3
  Name:            ShowDefaultText
  Type:            REG_DWORD
  Data:            0

b) On a machine that didn't have DDC, I downloaded the files (same as what I already installed from, but used these newer downloaded files to install DDC.   No HKLM\software\ddc key.

Do I have to add all the items in #13 for it to work if I want to change text?

**Comment by gianluigiloffreda** [comment]
Take a look at WOW64 folder. 


Inviato da iPhone

**Comment by TerraD** [comment]
Don't know what DDC is... As I understood @bduffek WOW64 folder is not important, it's ok to just add once those entries:

Basically DragDropConfirm can run without any key, but then there is no way to configure it. I added the registry keys myself. 

If you want to give it a try you can you my .reg file: https://web.tresorit.com/l/bAHyR#JFrSLzpwZH5Wvnr3hF50VA , This file will be working Win7..Win10. Download and Double-click on the file will insert the keys needed into the registry. You need to have rights to manipulate the the registry to do so. 

If you do not have rights to manipulate the registry it becomes complicated: Either ask someone who knows how to do or use google!

**Comment by feetsdr** [comment]
DDC - DragDropConfirm. Sorry, trying to save electrons : ) 

@TerraD Thanks for that file.  I was scratching my head though.  I'm a local admin on this PC.  I can run your .reg file, then try to move.  The message box comes up.

I then run my reg file.  No message.  I'll use yours and change the text, but scratching my head what I am doing wrong.

FINALLY! ItemText... I had gibberish.  Change that to &Move here and now my script works.  It's just a text field, right?

But I don't ever get prompted with that text.

ARGHH! I hate coding : )   You guys are great to have the patience / understanding to deal with small errors making a big difference!
